config.ru

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
This file is generated by rails, so we ignore to check with rubocop in `.rubocop.yml`. I would like to keep this file as is.